data/update.py

```diff
@@ -204,14 +204,10 @@ def gather_subdomains(gatherer, command):
       "--output=%s" % gatherer_output,
       "--parents=%s" % GATHER_PARENTS,
       "--sort",
+      "--force",
```

Comment:
Noticed a bug while doing this -- by leaving off the `--force` flag during gathering, this was reusing cached data from Censys.io. This means our Censys hostname seed data was stale for a while. (The scans of each domain's current compliance status was properly checked each scan.)